modules/openapi-generator/src/test/java/org/openapitools/codegen/java/JavaClientDeepObjectTest.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2018 OpenAPI-Generator Contributors (https://openapi-generator.tech)
+ * Copyright 2018 SmartBear Software
```

Comment:
Will remove this line in one of my upcoming PRs

Ref: https://opensource.stackexchange.com/questions/7300/copyright-notice-in-the-file-header-apache-v2-license